test/NonSilo.Tests/General/UtilsTests.cs

```diff
@@ -27,15 +27,15 @@ public void ToGatewayUriTest()
         {
             var ipv4 = new IPEndPoint(IPAddress.Any, 11111);
             var uri = ipv4.ToGatewayUri();
-            Assert.Equal("gwy.tcp://0.0.0.0:11111/0", uri.ToString());
```

Comment:
This represents a change of behavior that I should have picked up while reviewing this originally